src/encoded/types/ingestion_submission.py

```diff
@@ -17,3 +20,23 @@ class IngestionSubmission(Item, SnovaultIngestionSubmission):
     item_type = "ingestion_submission"
     schema = load_schema("encoded:schemas/ingestion_submission.json")
     embedded_list = []
+
+    @calculated_property(schema=display_title_schema)
+    def display_title(
+        self,
+        request: Request,
+        parameters: Optional[dict] = None
+    ) -> str:
+        if isinstance(parameters, dict) and _to_bool(parameters.get("validate_only")) is True:
+            # TODO: Look into why validate_only is sometimes a bool and sometimes a string.
+            return f"Validation:{self.uuid}"
+        return f"Submission:{self.uuid}"
```

Comment:
I would consider using something different from the UUID, you could use something like `Validation run for {filename} on {date}`?